roles/kubernetes/preinstall/tasks/dhclient-hooks-undo.yml

```diff
@@ -3,19 +3,24 @@
 # These tasks will undo changes done by kargo in the past if needed (e.g. when upgrading from kargo 2.0.x
 # or when changing resolvconf_mode)
 
+- name: Check if dhclient conf file exists
+  stat: path={{dhclientconffile}}
+  register: dhclientconf_stat
+
 - name: Remove kargo specific config from dhclient config
   blockinfile:
     dest: "{{dhclientconffile}}"
     state: absent
     backup: yes
     follow: yes
     marker: "# Ansible entries {mark}"
+  when: dhclientconf_stat.stat.exists
```

Comment:
_stat looks like a silly varname when you call $smth_stat.statnew patchset.. not sure why the previous failed(